server/render/index.js

```diff
@@ -55,12 +55,16 @@ export function render( element, key = JSON.stringify( element ) ) {
 			markupCache.set( key, context );
 		}
 		const rtsTimeMs = Date.now() - startTime;
+		debug( 'Server render time (ms)', rtsTimeMs );
 
 		if ( rtsTimeMs > 15 ) {
-			// We think that renderToString should generally
-			// never take more than 15ms. We're probably wrong.
+			// legacy stat from when we only had very simple server-renders
 			bumpStat( 'calypso-ssr', 'loggedout-design-over-15ms-rendertostring' );
```

Comment:
Do we want to rename this for consistency with the new stat (and since it's not specific to design anymore)? If we did, I figure the old line would end and a new one would start in the same point(?)